DigitalLearningSolutions.Web/ViewModels/Common/ViewComponents/LogoViewModel.cs

```diff
@@ -1,14 +1,14 @@
-﻿namespace DigitalLearningSolutions.Web.ViewModels.Common.ViewComponents
-{
-    using DigitalLearningSolutions.Data.Models;
-
-    public class LogoViewModel
-    {
-        public readonly Logo? Logo;
-
-        public LogoViewModel(Logo? logo)
-        {
-            Logo = logo;
-        }
-    }
-}
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Common.ViewComponents
+{
+    using global::DigitalLearningSolutions.Data.Models;

```

Comment:
Think this is a knock-on effect from the weird namespaces. Can we also check that the line endings here are CRLF, as it looks like they may have changed from before (likely was an old file that wasn't CRLF already).